backend/src/plugins/authorization/authorization.plugin.ts

```diff
@@ -14,6 +14,8 @@ import {
   user as userService,
 } from '~/services/services';
 
+const TOKEN_NULL_VALUE = 'null';
```

Comment:
how we can have a string null there? 
we will throw an error on the `decoded` step in this case. So we can avoid this check

Reply:
Yes, but the error message would be `Invalid JWT`, which is not a user-friendly message. And I checked with no token, the token was a string null :thinking:

Reply:
you can catch this error and throw a custom one 👍